packages/dev/test/RelayServer.test.ts

```diff
@@ -7,7 +7,7 @@ import sinon from 'sinon'
 import sinonChai from 'sinon-chai'
 import chaiAsPromised from 'chai-as-promised'
 
-import { GSNConfig } from '@opengsn/relayclient/GSNConfigurator'
+import { GSNConfig } from '@opengsn/provider/dist/GSNConfigurator'
```

Comment:
huh?

Reply:
It failed to import and `tsc` is not run for test files (executed inside `ts-node`) so it just went on with `GSNConfig` being `any`.

It is easy to make 'tsc' run for test files but will try to output them somewhere, not sure how useful is that. You won't need to run tests to see compilation errors, so probably will add `tsc` for tests later.